pkg/converter/types.go

```diff
@@ -66,6 +66,10 @@ type PackOption struct {
 	Compressor string
 	// OCIRef enables converting OCI tar(.gz) blob to nydus referenced blob.
 	OCIRef bool
+	// AlignedChunk aligns uncompressed data chunks to 4K, only for RAFS V5.
```

Comment:
Why do we need chunks to be aligned by 4K for rafs v5?

Reply:
It's compatible for `nydusify convert --aligned-chunk`.

Reply:
I think we don't have to expose the parameter `aligned-chunk`. Rafs v5 does not have to be aligned and rafs v6 must be aligned. Perhaps, the parameter can be removed both from nydusify and nydus-image

Reply:
We still need to keep the compatibility, at least it doesn't have any broken changes for minor release version, right?

Reply:
Um. there is no compatibility problem no matter whether aligned-chunk is exposed or not.

Reply:
The converter package will be used in nydusify v2.2.